action_tutorials/action_tutorials_cpp/src/fibonacci_action_client.cpp

```diff
@@ -42,6 +48,63 @@ class FibonacciActionClient : public rclcpp::Node
       this->get_node_waitables_interface(),
       "fibonacci");
 
+    auto on_set_parameter_callback =
+      [](std::vector<rclcpp::Parameter> parameters) {
```

Comment:
include `<vector>`

Reply:
Thanks. Fix it at https://github.com/ros2/demos/pull/709/commits/ffc65d6c050e4550e4287baa6ebad0b05b76e9de